src/model/analysis.ts

```diff
@@ -24,11 +24,11 @@ import {Warning} from './warning';
 
 
 // A regexp that matches paths to external code.
-// TODO(rictic): Make this extensible (polymer.json?).
-// Note that we match directories named exactly `build`, but will match any
-// directory name prefixed by `bower_components` or `node_modules`, in order to
-// ignore `polymer install`'s variants, which look like bower_components-foo
-const MATCHES_EXTERNAL = /(^|\/)(bower_components|node_modules|build($|\/))/;
+// TODO(rictic): Make this part of the URL Resolver.
+// Note that we will match any directory name prefixed by `bower_components` or
+// `node_modules` in order to ignore `polymer install`'s variants, which look
+// like bower_components-foo
+const MATCHES_EXTERNAL = /(^|\/)(bower_components|node_modules($|\/))/;
```

Comment:
why did we have "build" in there anyway?

Reply:
It's a common output dir name that was causing issues. Given that this wasn't configurable it was supposed to be a quick stopgap